specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2021-03-25-preview/examples/Machines_CreateOrUpdate.json

```diff
@@ -0,0 +1,73 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "machineName": "myMachine",
+    "api-version": "2021-03-25-preview",
+    "parameters": {
+      "location": "eastus2euap",
+      "properties": {
+        "vmId": "b7a098cc-b0b8-46e8-a205-62f301a62a8f",
+        "locationData": {
+          "name": "Redmond"
+        },
+        "clientPublicKey": "string",
+        "privateLinkScopeResourceId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/privateLinkScopes/privateLinkScopeName"
+      },
+      "identity": {
+        "type": "SystemAssigned"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/machines/myMachine",
+        "name": "myMachine",
+        "location": "eastus2euap",
+        "tags": null,
+        "identity": {
+          "type": "SystemAssigned",
+          "principalId": "string",
+          "tenantId": "string"
+        },
+        "type": "Microsoft.HybridCompute/machines",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "agentVersion": null,
+          "status": null,
+          "lastStatusChange": null,
+          "errorDetails": null,
+          "displayName": null,
+          "machineFqdn": null,
+          "vmUuid": null,
+          "osSku": null,
+          "domainName": null,
+          "adFqdn": null,
+          "dnsFqdn": null,
+          "osName": null,
+          "osVersion": null,
+          "osProfile": {
+            "computerName": null
+          },
+          "vmId": "b7a098cc-b0b8-46e8-a205-62f301a62a8f",
+          "locationData": {
+            "name": "Redmond",
+            "city": "redmond",
+            "district": null,
+            "countryOrRegion": "usa"
+          },
+          "clientPublicKey": "string",
+          "parentClusterResourceId": "{AzureStackHCIResourceId}",
```

Comment:
>"parentClusterResourceId": "{AzureStackHCIResourceId}" [](start = 10, length = 54)

Just wondering if this is ok or is the value supposed to be null?, reason I ask is because parentClusterResourceId is not present in the request.

Reply:
added